EasyPost.Tests/ErrorTest.cs

```diff
@@ -1,51 +1,9 @@
-﻿using System;
```

Comment:
Issue: Put this whole file back (multiple tests). Not testing errors is not acceptable.

I believe the error itself changed recently due to some changes made in the API. Feel free to look at some of the other libs for an example as they got their error tests updated when we switched fixtures.

Reply:
Error unit tests will be revamped when exception types and handling is refactored later in the rewrite process.

Reply:
Although that is true, we shouldn't be ripping out tests. The tests should remain, we should tweak them to match the current API behavior, and then when we refactor the error handling soon we can also revisit these error tests. 

Please put the tests back (and tweak values as necessary) so that we can ensure behavior matches our expectations from PR to PR.

Reply:
Resolved.